<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-DK86T691" ignored="0" name="用户管理系统测试用例" tests="7" failures="7" timestamp="21 四月 2019 14:12:10 GMT" time="3.899" errors="0">
  <testcase name="loginFalse" time="1.817" classname="com.course.cases.LoginTest">
    <failure type="java.lang.IllegalStateException" message="Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.">
      <![CDATA[java.lang.IllegalStateException: Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:216)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:401)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.LoginTest.getResult(LoginTest.java:74)
	at com.course.cases.LoginTest.loginFalse(LoginTest.java:55)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginFalse -->
  <testcase name="getUserInfo" time="0.918" classname="com.course.cases.GetUserInfoTest">
    <failure type="java.lang.IllegalStateException" message="Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.">
      <![CDATA[java.lang.IllegalStateException: Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:216)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:401)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.GetUserInfoTest.getJsonResult(GetUserInfoTest.java:57)
	at com.course.cases.GetUserInfoTest.getUserInfo(GetUserInfoTest.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- getUserInfo -->
  <testcase name="addUser" time="0.919" classname="com.course.cases.AddUserTest">
    <failure type="java.lang.IllegalStateException" message="Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.">
      <![CDATA[java.lang.IllegalStateException: Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:216)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:401)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.AddUserTest.getResult(AddUserTest.java:62)
	at com.course.cases.AddUserTest.addUser(AddUserTest.java:31)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- addUser -->
  <testcase name="updateUserInfo" time="0.919" classname="com.course.cases.UpdateUserInfoTest">
    <failure type="java.lang.IllegalStateException" message="Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.">
      <![CDATA[java.lang.IllegalStateException: Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:216)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:401)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.UpdateUserInfoTest.getResult(UpdateUserInfoTest.java:68)
	at com.course.cases.UpdateUserInfoTest.updateUserInfo(UpdateUserInfoTest.java:27)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- updateUserInfo -->
  <testcase name="getUserListInfo" time="0.88" classname="com.course.cases.GetUserInfoListTest">
    <failure type="java.lang.IllegalStateException" message="Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.">
      <![CDATA[java.lang.IllegalStateException: Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:216)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:401)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.GetUserInfoListTest.getJsonResult(GetUserInfoListTest.java:63)
	at com.course.cases.GetUserInfoListTest.getUserListInfo(GetUserInfoListTest.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- getUserListInfo -->
  <testcase name="deleteUser" time="0.88" classname="com.course.cases.UpdateUserInfoTest">
    <failure type="java.lang.IllegalStateException" message="Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.">
      <![CDATA[java.lang.IllegalStateException: Invalid use of SingleClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.impl.conn.SingleClientConnManager.getConnection(SingleClientConnManager.java:216)
	at org.apache.http.impl.conn.SingleClientConnManager$1.getConnection(SingleClientConnManager.java:190)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:401)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.UpdateUserInfoTest.getResult(UpdateUserInfoTest.java:68)
	at com.course.cases.UpdateUserInfoTest.deleteUser(UpdateUserInfoTest.java:44)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- deleteUser -->
  <testcase name="loginTrue" time="3.829" classname="com.course.cases.LoginTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connection to http://localhost:8889 refused">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connection to http://localhost:8889 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.LoginTest.getResult(LoginTest.java:74)
	at com.course.cases.LoginTest.loginTrue(LoginTest.java:41)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 22 more
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginTrue -->
</testsuite> <!-- 用户管理系统测试用例 -->
